src/config/settings.js

```diff
@@ -26,14 +26,20 @@ function load() {
 const defaultSettings = {
   kubernetes: {
     version: "v1.19.2"
+  },
+  minikube: {
+    allocations: {
+      memoryInGB: "2",
+      numberCPUs: 2,
+    }
```

Comment:
Can you please move these settings to the Kubernetes area. Minikube is just an implementation detail.

Reply:
sure

Reply:
I still want to keep these fields (and maybe the checkbox fields) out of k8s.vue to keep the unit tests simple.
k8s.vue knows too much about other parts of the system, and the less mocking we need to do, the better. Separation of concerns, and all that.